week-3/todo-list/script.js

```diff
@@ -1,25 +1,33 @@
-function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
-}
-
-// These are the same todos that currently display in the HTML
-// You will want to remove the ones in the current HTML after you have created them using JavaScript
-let todos = [
-  { task: "Wash the dishes", completed: false },
-  { task: "Do the shopping", completed: false },
-];
-
-populateTodoList(todos);
+let list = document.getElementById("todo-list");
+const addTodo = document.querySelector("#add");
+const todos = [];
 
-// This function will take the value of the input field and add it as a new todo to the bottom of the todo list. These new todos will need the completed and delete buttons adding like normal.
-function addNewTodo(event) {
+function addNewTodo() {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
-  // Write your code here... and remember to reset the input field to be blank after creating a todo!
+  const input = document.querySelector('input');
+  if (input.value != "") {
+    const li = document.createElement("li");
+    list.appendChild(li);
+    li.innerHTML = input.value;
+    li.classList.add('list-item');
+    const span = document.createElement("span");
+    const htmlContent = '<i id="check" onclick="this.parentNode.parentNode.style.textDecoration=`line-through`;" ></i>'
```

Comment:
This works well, but I'm curious, my instinct was that if I clicked the button again, it would un-strike-through - can you think how you'd implement that?

Also, I think the project brief was that the icon would change when it's crossed out - can you make that work too?